tests/RefreshDatabase.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+trait RefreshDatabase
+{
+    /**
+     * Refresh a conventional test database.
+     */
+    protected function refreshDatabase()
```

Comment:
isn't this similar to what we do in https://github.com/DoSomething/rogue/blob/master/app/Console/Commands/ClearDB.php